src/Microsoft.ML.HalLearners.StaticPipe/Microsoft.ML.HalLearners.StaticPipe.csproj

```diff
@@ -7,6 +7,7 @@
   <ItemGroup>
     <ProjectReference Include="..\Microsoft.ML.Data\Microsoft.ML.Data.csproj" />
     <ProjectReference Include="..\Microsoft.ML.HalLearners\Microsoft.ML.HalLearners.csproj" />
+    <ProjectReference Include="..\Microsoft.ML.StaticPipe\Microsoft.ML.StaticPipe.csproj" />
```

Comment:
>StaticPipe [](start = 71, length = 10)

why we have this?

Reply:
Without this `VectorWhiteningStaticExtensions.cs`  would complain of missing references.

Note previously the static extensions were part of the `Microsoft.ML.Data`  assembly.  In this PR we moved them to a separate assembly `Microsoft.ML.StaticPipe`

---
In reply to: [244826453](https://github.com/dotnet/machinelearning/pull/1930#discussion_r244826453) [](ancestors = 244826453)

Reply:
Make sense if this StaticPipe project

---
In reply to: [244829980](https://github.com/dotnet/machinelearning/pull/1930#discussion_r244829980) [](ancestors = 244829980,244826453)